tests/func/test_pipeline.py

```diff
@@ -98,6 +99,29 @@ def test_dot_commands(self):
         self.assertEqual(ret, 0)
 
 
+def test_disconnected_stage(tmp_dir, dvc, caplog):
+    tmp_dir.dvc_gen({"base": "base"})
+
+    dvc.add("base")
+    dvc.run(deps=["base"], outs=["derived1"], cmd="echo derived1 > derived1")
+    dvc.run(deps=["base"], outs=["derived2"], cmd="echo derived2 > derived2")
+    final_stage = dvc.run(
+        deps=["derived1"], outs=["final"], cmd="echo final > final"
+    )
+
+    args = ["pipeline", "show", "--outs", "--dot", final_stage.path]
+    with tmp_dir.chdir():
```

Comment:
You are already in tmp_dir, so no need to chdir. Or am I missing something here?

Reply:
That is correct! I didn't know about that. Updating.